web/html/src/manager/virtualization/HypervisorCheck.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import * as React from 'react';
+
+import { Messages } from 'components/messages';
+import { Utils as MessagesUtils } from 'components/messages';
+
+type Props = {
+    hypervisor: string,
+    foreignEntitled: boolean,
+};
+
+export function HypervisorCheck(props: Props) {
+    if (props.foreignEntitled) {
+        return null;
+    }
+
+    const virtMissing = props.hypervisor.includes("'virt' __virtual__ returned False");
+    if (virtMissing) {
+        const virtWarning = MessagesUtils.error(t("Please install libvirt python module."));
+        return <Messages items={virtWarning}/>;
+    }
+
+    const warning = props.hypervisor === "" ?
+        MessagesUtils.warning(t("Neither KVM nor Xen is running. Ensure they are installed and the kernel modules are loaded.")) :
```

Comment:
Just to make sure: is this intended to be a `warning` message AKA the user should be aware of it but still he can continue working with the current situation, or it is more an `error` message AKA the user should stop and fix it if he wants to continue working?

Reply:
in most cases he should fix first... for KVM however it could work in user-mode only though slower